velox/exec/tests/utils/LocalExchangeSource.cpp

```diff
@@ -38,38 +39,50 @@ class LocalExchangeSource : public exec::ExchangeSource {
 
   folly::SemiFuture<Response> request(
       uint32_t maxBytes,
-      uint32_t /*maxWaitSeconds*/) override {
+      uint32_t maxWaitSeconds) override {
     ++numRequests_;
 
     auto promise = VeloxPromise<Response>("LocalExchangeSource::request");
     auto future = promise.getSemiFuture();
```

Comment:
can we not just add a .onTimeout() callback to this future to trigger the lambda below?